### ISSUE TITLE
local.ini not being used

### ISSUE THREAD

**Issue by abulte** [klaemo/docker-couchdb#36]
Hi,

I have this simple Dockerfile, as per the doc, to inject my own configuration. I'm building/running through Docker Compose with no volume (not interested in volumes at this point).

```
FROM klaemo/couchdb
COPY local.ini /usr/local/etc/couchdb/
```

Problem is: the configuration specified in local.ini does not seem to be used. For example, this won't change the running port:

```
[httpd]
port = 5985
```

Any ideas? Thanks!

**Comment by abulte** [comment]
I can confirm by querying `_config` (still discovering CouchDB stuff right now ;-)) that my local.ini is not used.

**Comment by klaemo** [comment]
Hey there!
I just tried it and it works for me. My `Dockerfile` looks exactly like yours.
What does your `docker-compose.yml` look like? Maybe there's something missing.

**Comment by abulte** [comment]
My docker-compose.yml couldn't be simpler ;-)

```
couchdb:
    build: .
    ports:
        - 5984:5984
```

**Comment by klaemo** [comment]
You have to change the ports to `5985:5985`. The last one has to be `5985`, because that's where you expose couchdb now with the changed config (the first von can be anything, but it's easier if they are the same).
Also be sure to **really** rebuild your services, otherwise the changes don't get picked up.

**Comment by abulte** [comment]
Ok my bad, I was not clear about what I want to achieve.

I'm trying to setup CORS support in CouchDB. I only changed the port value as an example, to see if the config was used. After changing, my instance was still accessible on 5984 which led me to the fact the the local.ini was not used.

As for the "really rebuild" part, you were right! I just did a `docker-compose build --no-cache` and with the no-cache option it works...

Thanks for your help!
